packages/syft/src/syft/service/worker/worker_image.py

```diff
@@ -91,6 +101,10 @@ class SyftWorkerImage(SyftObject):
     image_hash: Optional[str]
     created_at: DateTime = DateTime.now()
     created_by: SyftVerifyKey
+    source_file: Optional[str]
+    full_tag: Optional[str]
```

Comment:
NIT: since already have a SyftWorkerTag, we could avoid creating full_tag attribute?

Reply:
to retrieve the full tag, we could do
image_tag.full_tag

Reply:
Done. Removed `full_tag` property and change `SyftWorkerImageTag` to `SyftWorkerImageIdentifier`